guides/doc-Provisioning_Guide/topics/proc_configuring_iPXE_to_reduce_provisioning_times.adoc

```diff
@@ -256,6 +287,22 @@ if exists user-class and option user-class = "iPXE" {
 ----
 +
 If you use an isolated network, use a {SmartProxyServer} URL with TCP port `8000`, instead of the URL of {ProjectServer}.
+. In creating a system entry to build it using iPXE workflow,
+ensure selecting the following as the PXELoader for BIOS and UEFI-based builds in the *Operating System* tab of the host:
++
+For BIOS Based systems, PXELoader can be selected as *None* or *iPXE Embedded*.
++
+For UEFI Based systems, PXELoader must be selected as *None*.
++
+iPXE keeps looping while configuring it to work with the Foreman discovery method for UEFI systems.
+To avoid it, enter the following commands:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# cp /usr/share/ipxe/ipxe.lkrn /var/lib/tftpboot/
+# cp /usr/share/ipxe/undionly.kpxe /var/lib/tftpboot/undionly-ipxe.0
+# restorecon -RvF /var/lib/tftpboot/
+----
```

Comment:
I don't think this is true, what prevents the looping is the PXELoader option, the way I understand the BZ.
Additionally, you're covering these commands at the top.

Reply:
Right. The cherry-picking commit has covered PXELoader and didn't repeat the step. :+1: